include/osw_hal.h

```diff
@@ -23,6 +23,9 @@ using std::string;
 // enum for user space button handling
 enum Button { BUTTON_1 = 0, BUTTON_2 = 1, BUTTON_3 = 2 };
 
+#define SCREEN_WIDTH 240
```

Comment:
This is already defined in config.h 
```
#define DISP_W 240
#define DISP_H 240
```
consider using:
hal->gfx()->getHeight() 
hal->gfx()->getWidth()